tests/api.c

```diff
@@ -15925,6 +15924,429 @@ static int test_wc_curve25519_export_key_raw_ex (void)
 #endif
     return 0;
 } /* end of test_wc_curve25519_export_key_raw_ex */
+/*
+ * Testing wc_curve25519_make_key
+ */
+static int test_wc_curve25519_make_key (void) 
+{
+    int ret = 0;
+#if defined(HAVE_CURVE25519)
+    WC_RNG           rng;
+    curve25519_key  key;
+    int           keysize;
```

Comment:
Can you line these variables up vertically?  If we are going to space the variables over from their type, we may as well line them up.  Also, double check other variable declaration sections in other test functions as well for alignment.